GeneratorInterface/RivetInterface/plugins/HTXSRivetProducer.cc

```diff
@@ -52,8 +49,8 @@ class HTXSRivetProducer : public edm::one::EDProducer<edm::one::WatchRuns, edm::
   edm::EDGetTokenT<edm::HepMCProduct> _hepmcCollection;
   edm::EDGetTokenT<LHERunInfoProduct> _lheRunInfo;
 
-  Rivet::AnalysisHandler _analysisHandler;
-  Rivet::HiggsTemplateCrossSections* _HTXS;
+  Rivet::AnalysisHandler* _analysisHandler = nullptr;
```

Comment:
It would be better if these were `std::unique_ptr<Rivet::AnalysisHandler>` and `std::unique_ptr<Rivet::HiggsTemplateCrossSections>`